### PR TITLE
modifed code to remove common electrodes

### PR DESCRIPTION
**Pull request by hvgazula**
i.e. we replace missing electrodes with np.nan

### CODE REVIEW

**Review by zkokaja — COMMENTED** [review]
Looks good to me! It would be nice to have single line comments explaining 7170 in case we forget what it means. A comment for put_signals_into_array function. And a comment explaining the except code when reading the header

File: code/tfspkl_utils.py [hunk 1]
```diff
@@ -193,8 +190,15 @@ def get_electrode_labels(conversation_dir):
     if not os.path.exists(header_file):
         return
```
Comment (zkokaja) [inline code/tfspkl_utils.py:191]: Should this be an error? What happens downstream if header file doesn't exist?

Comment (hvgazula) [reply code/tfspkl_utils.py:191]: Really redundant if you ask me. The header almost always exists. :)

**Comment by hvgazula** [comment]
> Looks good to me! It would be nice to have single line comments explaining 7170 in case we forget what it means. A comment for put_signals_into_array function. And a comment explaining the except code when reading the header

will do

**Pull request closed without merging**